app/views/items/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="page-title"><%= @merchant.name%></div>
+<br>
+<h2>List of Items</h2>
+<% @merchant.items.each do |item| %>
+  <%= link_to item.name, merchant_item_path(@merchant, item) %>
```

Comment:
good job with using the advanced routing! kinda tricky with having to pass 2 ids as params.